security/comply/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        strongdm comply 1.4.1 v
+
+description         Compliance automation framework, focused on SOC2
+
+long_description    Comply is a SOC2-focused compliance automation tool. It \
+                    includes: a Policy Generator: a markdown-powered document \
+                    pipeline for publishing auditor-friendly policy documents, \
+                    Ticketing Integration: to automate compliance throughout \
+                    the year via your existing ticketing system, and SOC2 \
+                    Templates: open source policy and procedure templates \
+                    suitable for satisfying a SOC2 audit.
+
+categories          security
+license             Apache-2
+platforms           darwin
+
+maintainers         {gmail.com:herby.gillot @herbygillot} openmaintainer
+
+checksums           rmd160  3cd4410085580544fde44990362b3d6d477d3945 \
+                    sha256  83b659fc77f6be5ead4f8355216bff74aef5b7e607ad72d3b7633e6052a128df \
+                    size    2826455
+
+depends_build       port:go
+
+installs_libs       no
+use_configure       no
+use_parallel_build  no
+
+set comply_example_dir "${prefix}/share/${name}/example"
+
+build {
+    set gopath [exec go env GOPATH]
+    set env(PATH) $env(PATH):${gopath}/bin/
+    file mkdir ${gopath}/src/github.com/strongdm
+    file link ${gopath}/src/github.com/strongdm/${name} ${worksrcpath}
+    system -W "${worksrcpath}" "${build.cmd} VERSION=${version} -w"
+}
```

Comment:
This block is different than how we would usually do this. 

Normally, you would set up the environment variables using a  ```build.env-append``` syntax, and perform the ```mkdir``` and ```link``` if needed in a ```pre-build``` block.

Then just let the normal build system run the build command.

Reply:
Gotcha.  I think one of the reasons why I did this was because the value of `GOPATH` was slightly different inside the `build` block as compared to if set in the main body of the `Portfile`.